src/components/containers/ArticleLists/ArticleLists.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Item, Icon } from 'semantic-ui-react';
+
+// components
+import Title from '../../presentationals/Title/Title';
+
+const f16 = {
+  fontSize: '16px',
+};
+
+class ArticleLists extends Component {
+  render() {
+    const { articles } = this.props;
+
+    return (
+      <div className="post-lists">
+        {articles.length === 0 ? (
+          <p>You have no publications</p>
+        ) : (
+          articles.map(article => (
+            <Item.Group key={article.id}>
+              <Item>
+                <Item.Image src="https://react.semantic-ui.com/images/wireframe/image.png" />
+
+                <Item.Content>
+                  <Title content={article.title} className="title-article-lg" />
+                  <Item.Description
+                    style={{
+                      marginBottom: 'auto',
+                    }}
+                  >
+                    <p
+                      style={{
+                        fontSize: '18px',
+                        lineHeight: '22px',
+                        fontWeight: '200',
+                      }}
+                    >
+                      {article.description}
+                    </p>
+                  </Item.Description>
+                  <Item.Extra>
+                    <div
+                      style={{
+                        display: 'flex',
+                        justifyContent: 'space-between',
+                      }}
+                    >
+                      <p style={f16}>3 days ago</p>
+                      <div
+                        style={{
+                          display: 'flex',
+                        }}
+                      >
+                        <p style={f16}>2 mins read</p>
+                        <button
+                          type="button"
+                          style={{
+                            padding: 0,
+                            border: 0,
+                            background: 'transparent',
+                          }}
+                        >
+                          <Icon
+                            name="bookmark outline"
+                            style={{
+                              fontSize: '20px',
```

Comment:
Likewise here. Is there a reason you chose px units over rem here?

Reply:
I am converting these to rem values now